Binary files XFree86-3.3.5.orig/xc/doc/specs/XIE/XIEProto/ch8_xie.doc and XFree86-3.3.5/xc/doc/specs/XIE/XIEProto/ch8_xie.doc differ
diff -urN XFree86-3.3.5.orig/xc/programs/Xserver/hw/xfree86/common/xf86Config.c XFree86-3.3.5/xc/programs/Xserver/hw/xfree86/common/xf86Config.c
--- XFree86-3.3.5.orig/xc/programs/Xserver/hw/xfree86/common/xf86Config.c	Thu Jul 29 11:22:44 1999
+++ XFree86-3.3.5/xc/programs/Xserver/hw/xfree86/common/xf86Config.c	Thu Sep  2 19:42:40 1999
@@ -774,11 +774,11 @@
     }
     
     /*
-     * /etc/XF86Config
+     * /etc/X11/XF86Config
      */
     configPaths[++pcount] = (char *)xalloc(PATH_MAX);
 #if !defined(__QNX__) || defined(__QNXNTO__)
-    strcpy(configPaths[pcount], "/etc/XF86Config");
+    strcpy(configPaths[pcount], "/etc/X11/XF86Config");
 #else
     /* On QNX, we default to /etc/config/XF86Config.nid to 
      * keep config files separate for network setups
